gui/layers/_vectors_layer/model.py

```diff
@@ -0,0 +1,624 @@
+#!/usr/bin/env python
+# title           : properties.py
+# description     :class Vectors layer that defines properties
+# author          :bryant.chhun
+# date            :1/16/19
+# version         :0.0
+# usage           :
+# notes           :
+# python_version  :3.6
+
+from typing import Union
+
+import numpy as np
+
+from .._base_layer import Layer
+from .._register import add_to_viewer
+from gui._vispy.scene.visuals import Line as LinesNode
```

Comment:
This import should be a relative path

`from ..._vispy.scene.visuals import Line as LinesNode`